e2e/test/iothub/FileUploadE2ETests.cs

```diff
@@ -23,14 +23,6 @@ public class FileUploadE2ETests : E2EMsTestBase
         private const int FileSizeSmall = 10 * 1024;
         private const int FileSizeBig = 5120 * 1024;
 
-        [LoggedTestMethod]
```

Comment:
Just because the SDK method is marked as obsolete, doesn't mean a customer won't still use it and we need to ensure it is tested.

We shouldn't remove these tests.

Reply:
Yeah good call, I'll fix that.

Reply:
Also, please update the PR title because when I see updates for it, I have a small heart attack each time. :)

Reply:
Title updated, heart attack avoided. :)